clusters/sandbox.yaml

```diff
@@ -43,7 +43,7 @@ namespaces:
 - name: sandbox-metadata-controller
   owner: alphagov
   repository: verify-metadata-controller
-  branch: master
+  branch: proxy-node-mesh
```

Comment:
I think you might want to do the same for the verify-proxy-node repo too:

https://github.com/alphagov/verify-proxy-node/tree/proxy-node-mesh/ci

`ci/sandbox` only currently exists in that branch

Reply:
🙈